<?xml version="1.0" encoding="UTF-8" ?>
<launch>
	<!-- SETUP OF THE NODES AND CONFIGURATIONS -->
	
	<!-- The logger -->
	<node name="roburoc4_trajectory_logger" pkg="roburoc4" type="roburoc4_trajectory_logger" output="screen" />
	
	<!-- Datafolder for the logger -->
	<param name="/roburoc4_controllers/recorded_trajectories" value="~/recorded_trajecotries" />

</launch>
